aws/resource_aws_codebuild_source_credential.go

```diff
@@ -0,0 +1,130 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/codebuild"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceAwsCodeBuildSourceCredential() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsCodeBuildSourceCredentialCreate,
+		Read:   resourceAwsCodeBuildSourceCredentialRead,
+		Delete: resourceAwsCodeBuildSourceCredentialDelete,
+
```

Comment:
Can you please implement resource import support? More information on the required steps can be found in the [Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#adding-resource-import-support). This resource testing will likely require `ImportStateVerifyIgnore: []string{"token", "user_name"}` Thanks!

```suggestion
		Importer: &schema.ResourceImporter{
			State: schema.ImportStatePassthrough,
		},

```

Reply:
fixed in c13110c